tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -46,87 +49,117 @@ describe('<AddonMoreInfo />', () => {
     delete partialAddon.support_url;
     const root = render({ addon: partialAddon });
 
-    expect(root.supportLink).toEqual(undefined);
+    expect(root.find('.AddonMoreInfo-support-link')).toHaveLength(0);
   });
 
   it('renders the support link of an add-on', () => {
     const root = render();
+    const link = root.find('.AddonMoreInfo-support-link');
 
-    expect(root.supportLink.textContent).toEqual('Support Site');
-    expect(root.supportLink.tagName).toEqual('A');
-    expect(root.supportLink.href).toEqual('http://support.hampsterdance.com/');
+    expect(link).toIncludeText('Support Site');
+    expect(link).toHaveProp('href', 'http://support.hampsterdance.com/');
   });
 
   it('renders the version number of an add-on', () => {
     const root = render();
 
-    expect(root.version.textContent).toEqual('2.0.0');
-    expect(root.version.tagName).toEqual('DD');
+    expect(root.find('.AddonMoreInfo-version')).toHaveText('2.0.0');
   });
 
   it('renders the license and link', () => {
     const root = render();
 
-    expect(root.licenseHeader.textContent).toEqual('License');
-    expect(root.licenseLink.textContent).toEqual('tofulicense');
-    expect(root.licenseLink.tagName).toEqual('A');
-    expect(root.licenseLink.href).toEqual('http://license.com/');
+    expect(root.find('.AddonMoreInfo-license-title')).toHaveText('License');
+    expect(root.find('.AddonMoreInfo-license-link'))
+      .toHaveText('tofulicense');
+    expect(root.find('.AddonMoreInfo-license-link'))
+      .toHaveProp('href', 'http://license.com/');
   });
 
   it('does not render a privacy policy if none exists', () => {
     const partialAddon = { ...fakeAddon, has_privacy_policy: false };
     const root = render({ addon: partialAddon });
 
-    expect(root.privacyPolicyLink).toEqual(undefined);
+    expect(root.find('.AddonMoreInfo-privacy-policy-title'))
+      .toHaveLength(0);
+    expect(root.find('.AddonMoreInfo-privacy-policy-link'))
+      .toHaveLength(0);
   });
 
   it('renders the privacy policy and link', () => {
     const addon = { ...fakeAddon, has_privacy_policy: true };
     const root = render({ addon });
 
-    expect(findDOMNode(root.privacyPolicyLink).textContent).toEqual('Read the privacy policy for this add-on');
-    // TODO: Change this to an internal `<Link>` tag and use `expect().toBe`
-    // once https://github.com/mozilla/addons-frontend/issues/1828 is fixed.
-    expect(root.privacyPolicyLink.props.href).toContain('/addon/chill-out/privacy/');
+    expect(root.find('.AddonMoreInfo-privacy-policy-title'))
+      .toHaveText('Privacy Policy');
+    expect(root.find('.AddonMoreInfo-privacy-policy-link'))
+      .toHaveProp('children', 'Read the privacy policy for this add-on');
+    expect(root.find('.AddonMoreInfo-privacy-policy-link'))
+      .toHaveProp('href', '/addon/chill-out/privacy/');
+  });
+
+  it('does not render a EULA if none exists', () => {
+    const partialAddon = { ...fakeAddon, has_eula: false };
```

Comment:
This was only called `partialAddon` in the other test because one of its properties got deleted.

Reply:
True! There's also the fact that a lot of the tests rely on the default props in `fakeAddon` but I like the pattern of explicitly setting the props we're testing so it's visible the relationship. I'll quickly tidy up the tests...